src/modules/home/heroSlider.tsx

```diff
@@ -11,10 +11,10 @@ import {
 import Image from "next/image";
 
 const heroImages = [
-  { id: 1, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 1" },
-  { id: 2, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 2" },
-  { id: 3, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 3" },
-  { id: 4, imageSrc: "/images/hero-image.jpg", imageAlt: "group image 4" },
+  { id: 1, imageSrc: "/images/hero-image-1.jpg", imageAlt: "group image 1" },
+  { id: 2, imageSrc: "/images/hero-image-2.jpg", imageAlt: "group image 2" },
+  { id: 3, imageSrc: "/images/hero-image-3.jpg", imageAlt: "group image 3" },
```

Comment:
Hey @yashcrest can we please crop the picture frame in the background from hero-image-3? 🙈

Reply:
lol... sure. 
have cropped out the photo frame from hero-image-3  now 👍 .

Reply:
I have committed the changes to my branch but I don't see the changes here. Are you able to see it in your end? 
![Uploading Screenshot 2024-09-16 at 6.07.46 PM.png…]()